examples/example_config_files/isochrone_test.yaml

```diff
@@ -46,6 +46,13 @@ detectors:
             h: 0.1
             translate:
               z: 0.05
+        symmetry:
+          phi:
+            left: reflecting
+            right: fixed
+          r:
+            left: fixed
+            right: fixed
```

Comment:
No, we want to switch to define the symmetries. Not the boundary conditions.

So something like: 
```yaml
symmetries:
     mirror_symmetry: 
           phi: 180
```
Then, the boundary conditions will be determined from the symmetries internally
for both kind of grids, cylindrical and Cartesian, automatically.